SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -423,4 +275,19 @@ var splunkRumInitializeCalledTime = Date()
     @objc public class func isInitialized() -> Bool {
         return initialized
     }
+
+    /**
+       Updates the current location. The latitude and longitude will be appended to every span and event.
+     */
+    @objc public class func setLocation(latitude: Double, longitude: Double) {
+            setGlobalAttributes([Attribute.LOCATION_LONGITUDE_KEY: longitude])
+            setGlobalAttributes([Attribute.LOCATION_LATITUDE_KEY: latitude])
+    }
+
+    /**
+      Set up the deployment environment for RUM instance. This will be passed along as a span
+     */
+    @objc public class func deploymentEnvironment(environment: String) {
+            setGlobalAttributes([Attribute.DEPLOYMENT_ENVIRONMENT: environment])
+    }
```

Comment:
Deployment environment is only set at startup, so this method can be removed

Reply:
Removed